model/task_lifecycle.go

```diff
@@ -1017,7 +1021,20 @@ func UpdateDisplayTask(t *task.Task) error {
 		}
 	}
 
-	if hasFinishedTasks && hasUnfinishedTasks {
+	grip.Debug(message.Fields{
```

Comment:
should we keep this in?

Reply:
We put it in at the end of the last commit to monitor that this was working as we expected. That being said, the solution is much simpler now so I'm not opposed to removing it.